fdbserver/TagPartitionedLogSystem.actor.cpp

```diff
@@ -2614,6 +2614,8 @@ ACTOR Future<Void> TagPartitionedLogSystem::newRemoteEpoch(TagPartitionedLogSyst
 		req.tLogLocalities = localities;
 		req.tLogPolicy = logSet->tLogPolicy;
 		req.locality = remoteLocality;
+		TraceEvent("RemoteTLogRouterReplies", self->dbgid)
```

Comment:
This is in a loop, the same event may be throttled if there are too many (not sure about the threshold). A solution is to add "i" to the trace event name so events are different.

The same applies to other trace events in this PR.

Probably "InitLogRouter" is more accurate than "RemoteTLogRouterReplies". The "Replies" implies this is a reply, but actually is not.

Reply:
The naming follows the same convention as `ResolverReplies`, `CommitProxyReplies` and `GrvProxyReplies`. I once wanted to change them to `Initialize...`, but received opposite suggestion because people have tooling that already use those TraceEvent names. Our dashboard now relies on those TraceEvent too, so I'd keep the same naming format for consistency.

Regarding throttling, Tlog has same granularity as CommitProxies, and we haven't seen `CommitProxyReplies` being throttled. `TRACE_EVENT_THROTTLER_MSG_LIMIT` is set to 20000.